src/validations/html/slowparse.js

```diff
@@ -124,7 +124,10 @@ class SlowparseValidator extends Validator {
   }
 
   async _getRawErrors() {
-    const {Slowparse} = await System.import('../linters');
+    const {Slowparse} = await import(
+      /* webpackChunkName: 'linters' */
+      '../linters',
+    );
```

Comment:
I’m not sure why this doesn’t use `importLinters` like every other validation—seems like I [just missed it in the original PR](https://github.com/popcodeorg/popcode/pull/587)